app/services.py

```diff
@@ -84,12 +84,15 @@ async def create(self, model, exclude: tuple = ("id",)):
 
     async def update(self, model, exclude: tuple = ("id",), query=None):
         query = query or (self.table.c.id == model.id)
-        return await self.cursor.one(
+        res = await self.cursor.one(
             self.table.update(query)
             .values(model.dict(exclude=set(exclude)))
             .returning(self.table)
         )
+        await self.commit()
```

Comment:
do not commit inside service, use only outside so if we have more than 1 query to commit we will have tx rollback